types/react-dom/index.d.ts

```diff
@@ -17,7 +16,7 @@ import {
     DOMAttributes, DOMElement, ReactNode, ReactPortal
 } from 'react';
 
```

Comment:
You should add type `Text` declaration first.

Reply:
`Text` comes from the ambient declaration in `lib.dom.d.ts`:

![image](https://user-images.githubusercontent.com/13781/46564418-fa628400-c8bb-11e8-896a-d3de0e59fcfc.png)

If you’re seeing a type error, something might be misconfigured in your tsconfig file (for example, the `dom` lib might not be included).